-- phpMyAdmin SQL Dump
-- version 5.2.0
-- https://www.phpmyadmin.net/
--
-- Host: localhost:3306
-- Generation Time: May 07, 2025 at 07:23 AM
-- Server version: 8.0.30
-- PHP Version: 8.3.6

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
START TRANSACTION;
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;

--
-- Database: `pelatihan_data_science`
--

-- --------------------------------------------------------

--
-- Table structure for table `clan`
--

CREATE TABLE `clan` (
  `clan_acc` varchar(7) NOT NULL,
  `id` varchar(40) NOT NULL,
  `previous_id` tinytext CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci,
  `description` varchar(100) NOT NULL,
  `author` tinytext NOT NULL,
  `comment` longtext NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

--
-- Dumping data for table `clan`
--

INSERT INTO `clan` (`clan_acc`, `id`, `previous_id`, `description`, `author`, `comment`) VALUES
('CL00001', 'tRNA', NULL, 'tRNA clan', 'Gardner PP', 'The tRNA clan contains the RNA families tRNA and tmRNA. Homology between these families has been established in the published literature [1-5].'),
('CL00002', 'RNaseP', NULL, 'RNaseP clan', 'Gardner PP', 'The RNaseP clan contains the RNA families RNaseP_nuc, RNaseP_bact_a, RNaseP_bact_b, RNase_MRP and RNaseP_arch. Homology between these families has been established in the published literature [1-7].'),
('CL00003', 'SRP', NULL, 'SRP clan', 'Gardner PP; 0000-0002-7808-1213', 'The SRP clan contains the RNA families for Metazoa_SRP, Bacterial_small_SRP, Bacterial_large_SRP, Fungal_SRP, Plant_SRP, Protozoa_SRP, Archaea_SRP and Dictyostelium_SRP. Homology between these families has been established in the published literature [1-8].'),
('CL00004', 'Telomerase', NULL, 'Telomerase clan', 'Gardner PP', 'The Telomerase clan contains the RNA families Telomerase-vert, Telomerase-cil and Sacc_telomerase. Homology between these families has been established in the published literature [1-3].'),
('CL00005', 'U1', NULL, 'U1 clan', 'Gardner PP', 'The U1 clan contains the RNA families U1, U1_yeast and U11. Homology between these families has been established in the published literature [1-4].'),
('CL00006', 'U2', NULL, 'U2 clan', 'Gardner PP', 'The U2 clan contains the RNA families U2 and U12. Homology between these families has been established in the published literature [1-4].'),
('CL00007', 'U4', NULL, 'U4 clan', 'Gardner PP', 'The U4 clan contains the RNA families U4 and U4atac. Homology between these families has been established in the published literature [1-5].'),
('CL00008', 'U54', NULL, 'U54 clan', 'Gardner PP', 'The U54 clan contains the RNA families U54 and snoU54. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [1,2].'),
('CL00009', 'U6', NULL, 'U6 clan', 'Gardner PP', 'The U6 clan contains the RNA families U6 and U6atac. Homology between these families has been established in the published literature [1-4].'),
('CL00010', 'Hammerhead', NULL, 'Hammerhead clan', 'Gardner PP', 'The Hammerhead clan contains the RNA families Hammerhead_3 and Hammerhead_1. Homology between these families has been established in the published literature [1,2].'),
('CL00011', 'Glm', NULL, 'Glm clan', 'Gardner PP', 'The Glm clan contains the RNA families GlmZ_SraJ and GlmY_tke1. Homology between these families has been established in the published literature [1].'),
('CL00012', 'SAM', NULL, 'SAM clan', 'Gardner PP', 'The SAM clan contains the RNA families SAM, SAM-IV and SAM-I/IV. Homology between these families has been established in the published literature [1].'),
('CL00013', '7SK', NULL, '7SK clan', 'Gardner PP', 'The 7SK clan contains the RNA families 7SK and Arthropod_7SK. Homology between these families has been established in the published literature [1].'),
('CL00014', 'CRISPR-1', NULL, 'CRISPR-1 clan', 'Gardner PP', 'The CRISPR-1 clan contains the RNA families CRISPR-DR2, CRISPR-DR4, CRISPR-DR14, CRISPR-DR17, CRISPR-DR25, CRISPR-DR43 and CRISPR-DR66. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00015', 'CRISPR-2', NULL, 'CRISPR-2 clan', 'Gardner PP', 'The CRISPR-2 clan contains the RNA families CRISPR-DR5, CRISPR-DR7, CRISPR-DR63 and CRISPR-DR64. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00016', 'FinP-traJ', NULL, 'FinP-traJ clan', 'Gardner PP', 'The FinP-traJ clan contains the RNA families FinP and traJ_5. Homology between these families has been established in the published literature [1]. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH and PRC [2,3].'),
('CL00017', 'IRES1', NULL, 'IRES1 clan', 'Gardner PP', 'The IRES1 clan contains the RNA families IRES_HCV and IRES_Pesti. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH and PRC [1,2].'),
('CL00018', 'SCARNA3', NULL, 'SCARNA3 clan', 'Gardner PP', 'The SCARNA3 clan contains the RNA families SCARNA24 and SCARNA3. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH and PRC [1,2].'),
('CL00019', 'SCARNA4', NULL, 'SCARNA4 clan', 'Gardner PP', 'The SCARNA4 clan contains the RNA families SCARNA4 and SCARNA15. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00020', 'SL', NULL, 'SL clan', 'Gardner PP', 'The SL clan contains the RNA families SL1 and SL2. Homology between these families has been established in the published literature [1]. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [2,3].'),
('CL00021', 'SNORA2', NULL, 'SNORA2 clan', 'Gardner PP', 'The SNORA2 clan contains the RNA families SNORA16, SNORA2, snopsi28S-3327, snR189, snR46 and S_pombe_snR46. Homology between these families has been established in the published literature [1]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA2 and snR189, SNORA16 and snR46 [2].'),
('CL00022', 'SNORA3', NULL, 'SNORA3 clan', 'Gardner PP', 'The SNORA3 clan contains the RNA families SNORA3 and snR11. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA3 and snR11 [1].'),
('CL00023', 'SNORA4', NULL, 'SNORA4 clan', 'Gardner PP', 'The SNORA4 clan contains the RNA families SNORA4 and snR83. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA4 and snR83 [1].'),
('CL00024', 'SNORA5', NULL, 'SNORA5 clan', 'Gardner PP', 'The SNORA5 clan contains the RNA families SNORA5 and snR85. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA5 and snR85 [1].'),
('CL00025', 'SNORA7', NULL, 'SNORA7 clan', 'Gardner PP', 'The SNORA7 clan contains the RNA families SNORA7 and snR81. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA7 and snR81 [1].'),
('CL00026', 'SNORA8', NULL, 'SNORA8 clan', 'Gardner PP', 'The SNORA8 clan contains the RNA families SNORA8 and snR31. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA8 and snR31 [1].'),
('CL00027', 'SNORA9', NULL, 'SNORA9 clan', 'Gardner PP', 'The SNORA9 clan contains the RNA families SNORA9, snR33, snR43 and S_pombe_snR33. Homology between these families has been established in the published literature [1]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA9 and snR43, SNORA9 and snR33 [2].'),
('CL00028', 'SNORA13', NULL, 'SNORA13 clan', 'Gardner PP', 'The SNORA13 clan contains the RNA families SNORA13, snR35 and S_pombe_snR35. Homology between these families has been established in the published literature [1]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA13 and snR35 [2].'),
('CL00029', 'SNORA17', NULL, 'SNORA17 clan', 'Gardner PP', 'The SNORA17 clan contains the RNA families SNORA43 and SNORA17. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00030', 'SNORA20', NULL, 'SNORA20 clan', 'Gardner PP', 'The SNORA20 clan contains the RNA families SNORA20 and SNORA29. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00031', 'SNORA21', NULL, 'SNORA21 clan', 'Gardner PP', 'The SNORA21 clan contains the RNA families SNORA21, snR10 and S_pombe_snR10. Homology between these families has been established in the published literature [1]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA21 and snR10 [2].'),
('CL00032', 'SNORA27', NULL, 'SNORA27 clan', 'Gardner PP', 'The SNORA27 clan contains the RNA families SNORA27, SNORA26, snR42 and S_pombe_snR42. Homology between these families has been established in the published literature [1]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA27 and snR42 [2].'),
('CL00033', 'SNORA28', NULL, 'SNORA28 clan', 'Gardner PP', 'The SNORA28 clan contains the RNA families SNORA28, snopsi18S-841 and snR80. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA28 and snR80 [1].'),
('CL00034', 'SNORA35', NULL, 'SNORA35 clan', 'Gardner PP', 'The SNORA35 clan contains the RNA families SNORA50, SNORA54, SNORA35 and SNORA76. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00035', 'SNORA36', NULL, 'SNORA36 clan', 'Gardner PP', 'The SNORA36 clan contains the RNA families SNORA36, snR36, snR44 and S_pombe_snR36. Homology between these families has been established in the published literature [1]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA36 and snR36, SNORA36 and snR44 [2].'),
('CL00036', 'SNORA44', NULL, 'SNORA44 clan', 'Gardner PP', 'The SNORA44 clan contains the RNA families SNORA44, SNORA58, snR161 and snR9. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA44 and snR161, SNORA58 and snR9 [1].'),
('CL00037', 'SNORA48', NULL, 'SNORA48 clan', 'Gardner PP', 'The SNORA48 clan contains the RNA families SNORA48 and snR86. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA48 and snR86 [1].'),
('CL00038', 'SNORA52', NULL, 'SNORA52 clan', 'Gardner PP', 'The SNORA52 clan contains the RNA families SNORA66, snoR98, SNORA52, SNORA18, snoR80, snR49, snR5 and S_pombe_snR5. Homology between these families has been established in the published literature [1]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: U66 and snR49, SNORA52 and snR5 [2]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoR80 and SnR5, snoR98 and SnR5 [3].'),
('CL00039', 'SNORA56', NULL, 'SNORA56 clan', 'Gardner PP', 'The SNORA56 clan contains the RNA families SNORA56 and snR8. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA56 and snR8 [1].'),
('CL00040', 'SNORA62', NULL, 'SNORA62 clan', 'Gardner PP', 'The SNORA62 clan contains the RNA families SNORA62, snR65, snR3, snR82 and S_pombe_snR3. Homology between these families has been established in the published literature [1]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA62 and snR3, SNORA62 and snR65, SNORA62 and snR82 [2].'),
('CL00041', 'SNORA64', NULL, 'SNORA64 clan', 'Gardner PP', 'The SNORA64 clan contains the RNA families SNORA64 and snR37. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA64 and snR37 [1].'),
('CL00042', 'SNORA65', NULL, 'SNORA65 clan', 'Gardner PP', 'The SNORA65 clan contains the RNA families SNORA65, snR34 and snoR2. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA65 and snR34 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoR2 and U65&SnR34 [2].'),
('CL00043', 'SNORA74', NULL, 'SNORA74 clan', 'Gardner PP', 'The SNORA74 clan contains the RNA families SNORA74 and snR191. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA74 and snR191 [1]. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 [2].'),
('CL00044', 'SNORD12', NULL, 'SNORD12 clan', 'Gardner PP', 'The SNORD12 clan contains the RNA families SNORD12 and snR190. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD12 and snR190 [1].'),
('CL00045', 'SNORD15', NULL, 'SNORD15 clan', 'Gardner PP', 'The SNORD15 clan contains the RNA families SNORD15, snR75, snR13 and snoZ5. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD15 and snR13 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD15 and U15&SnR13, SNORD15 and SnR75 [2].'),
('CL00046', 'SNORD16', NULL, 'SNORD16 clan', 'Gardner PP', 'The SNORD16 clan contains the RNA families SNORD16 and snR87. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: U16 and snR87 [1].'),
('CL00047', 'SNORD18', NULL, 'SNORD18 clan', 'Gardner PP', 'The SNORD18 clan contains the RNA families SNORD18 and snoU18. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD18 and snoU18 [1].'),
('CL00048', 'SNORD19', NULL, 'SNORD19 clan', 'Gardner PP', 'The SNORD19 clan contains the RNA families SNORD19 and SNORD19B. Homology between these families has been established in the published literature [1]. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [2,3,4].'),
('CL00049', 'SNORD25', NULL, 'SNORD25 clan', 'Gardner PP', 'The SNORD25 clan contains the RNA families SNORD25 and snR56. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD25 and snR56 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD25 and U25&SnR56 [2].'),
('CL00050', 'SNORD26', NULL, 'SNORD26 clan', 'Gardner PP', 'The SNORD26 clan contains the RNA families SNORD26 and SNORD81. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [1,2].'),
('CL00051', 'SNORD29', NULL, 'SNORD29 clan', 'Gardner PP', 'The SNORD29 clan contains the RNA families SNORD36, SNORD29, snoZ223, SNORD38, snosnR69, snosnR61, snosnR71, SNORD78, snoR69Y, snR47 and snoU36a. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD29 and snR71, SNORD36 and snR47, SNORD38 and snR61 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoU36a and U36a&SnR47, snoR69Y and U29&SnR71, snoZ223 and U38ab&SnR61, snoR69Y and SnR69 [2].'),
('CL00052', 'SNORD30', NULL, 'SNORD30 clan', 'Gardner PP', 'The SNORD30 clan contains the RNA families SNORD30 and snoU30. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoU30 and U30 [1].'),
('CL00053', 'SNORD31', NULL, 'SNORD31 clan', 'Gardner PP', 'The SNORD31 clan contains the RNA families SNORD31, snoZ17, snR67 and snoR35. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD31 and snR67 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoR35 and U31&SnR67 [2]. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 [3].'),
('CL00054', 'SNORD33', NULL, 'SNORD33 clan', 'Gardner PP', 'The SNORD33 clan contains the RNA families SNORD33, snoZ196, SNORD51, snosnR55, snoMe18S-Um1356, snoMe28S-Am982, snR39, snR40 and snoR21b. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD33 and snosnR55 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoR21b and U32&SnR40, snoZ196 and U51&U32a&SnR39&SnR59 [2].'),
('CL00055', 'SNORD34', NULL, 'SNORD34 clan', 'Gardner PP', 'The SNORD34 clan contains the RNA families SNORD34 and snR62. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD34 and snR62 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD34 and U34&SnR62 [2].'),
('CL00056', 'SNORD35', NULL, 'SNORD35 clan', 'Gardner PP', 'The SNORD35 clan contains the RNA families SNORD35, snoZ161_228 and snR73. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD35 and snR73 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoZ161_228 and U35&SnR73 [2].'),
('CL00057', 'SNORD39', NULL, 'SNORD39 clan', 'Gardner PP', 'The SNORD39 clan contains the RNA families SNORD55, snoZ7, snoZ101, SNORD65, snoR77Y and snR77. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD65/HBII-135 and snR77 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoZ101 and U55&U39 [2].'),
('CL00058', 'SNORD41', NULL, 'SNORD41 clan', 'Gardner PP', 'The SNORD41 clan contains the RNA families SNORD57, SNORD41 and snR51. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: U57 and snR51, SNORD41 and snR51 [1].'),
('CL00059', 'SNORD43', NULL, 'SNORD43 clan', 'Gardner PP', 'The SNORD43 clan contains the RNA families SNORD43 and snR70. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD43 and snR70 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD43 and U43&SnR70 [2].'),
('CL00060', 'SNORD44', NULL, 'SNORD44 clan', 'Gardner PP', 'The SNORD44 clan contains the RNA families SNORD44 and snoZ102_R77. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoZ102_R77 and U44 [1].'),
('CL00061', 'SNORD46', NULL, 'SNORD46 clan', 'Gardner PP', 'The SNORD46 clan contains the RNA families SNORD46 and snR63. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD46 and snR63 [1].'),
('CL00062', 'SNORD49', NULL, 'SNORD49 clan', 'Gardner PP', 'The SNORD49 clan contains the RNA families SNORD49, snoZ112 and snoU49. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoU49 and U49 [1].'),
('CL00063', 'SNORD52', NULL, 'SNORD52 clan', 'Gardner PP', 'The SNORD52 clan contains the RNA families SNORD52, SNORD53, snoZ157 and snR78. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD52 and snR78 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoZ157 and U53, snoZ157 and U52&SnR78 [2].'),
('CL00064', 'SNORD58', NULL, 'SNORD58 clan', 'Gardner PP', 'The SNORD58 clan contains the RNA families SNORD58 and SNORD99. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [1,2].'),
('CL00065', 'SNORD59', NULL, 'SNORD59 clan', 'Gardner PP', 'The SNORD59 clan contains the RNA families snoZ159, SNORD59 and snosnR54. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD59 and snosnR54 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoZ159 and U59&SnR54 [2].'),
('CL00066', 'SNORD60', NULL, 'SNORD60 clan', 'Gardner PP', 'The SNORD60 clan contains the RNA families SNORD60, snoR1, snosnR48, snoMe28S-G3255 and sn2903. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD60 and snR48 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoR1 and SnR48 [2].'),
('CL00067', 'SNORD61', NULL, 'SNORD61 clan', 'Gardner PP', 'The SNORD61 clan contains the RNA families SNORD61, snoU61 and SNORD125. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoU61 and U61 [1].'),
('CL00068', 'SNORD62', NULL, 'SNORD62 clan', 'Gardner PP', 'The SNORD62 clan contains the RNA families SNORD62, snoR41 and snR41. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: U62 and snR41 [1].'),
('CL00069', 'SNORD74', NULL, 'SNORD74 clan', 'Gardner PP', 'The SNORD74 clan contains the RNA families SNORD79, SNORD113, SNORD74, snoR44_J54, snosnR64, snoMe28S-Cm2645, SNORD64, SNORD112 and SNORD109A. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD74 and snR64 [1]. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoR44_J54 and U74&SnR64, snoR44_J54 and U79 [2]. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 [3].'),
('CL00070', 'SNORD77', NULL, 'SNORD77 clan', 'Gardner PP', 'The SNORD77 clan contains the RNA families snosnR60_Z15 and SNORD77. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD77 and snR60 [1].'),
('CL00071', 'SNORD88', NULL, 'SNORD88 clan', 'Gardner PP', 'The SNORD88 clan contains the RNA families SNORD88, snR76 and snoR118. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD88 and snR76 [1]. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 [2].'),
('CL00072', 'SNORD96', NULL, 'SNORD96 clan', 'Gardner PP', 'The SNORD96 clan contains the RNA families SNORD96 and snR39B. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snR39B and snR39b [1].'),
('CL00073', 'SNORD100', NULL, 'SNORD100 clan', 'Gardner PP', 'The SNORD100 clan contains the RNA families snoR30 and SNORD100. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00074', 'SNORD101', NULL, 'SNORD101 clan', 'Gardner PP', 'The SNORD101 clan contains the RNA families SNORD101 and snoR60. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00075', 'SNORD105', NULL, 'SNORD105 clan', 'Gardner PP', 'The SNORD105 clan contains the RNA families SNORD105 and snoU105B. Homology between these families has been established in the published literature [1,2]. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [3,4,5].'),
('CL00076', 'SNORD110', NULL, 'SNORD110 clan', 'Gardner PP', 'The SNORD110 clan contains the RNA families SNORD110 and snoR14. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORD110 and snoR14 [1].'),
('CL00077', 'SNORND104', NULL, 'SNORND104 clan', 'Gardner PP', 'The SNORND104 clan contains the RNA families SNORND104 and snR58. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORND104 and snR58 [1].'),
('CL00078', 'snR30-U17', NULL, 'snR30-U17 clan', 'Gardner PP', 'The snR30-U17 clan contains the RNA families SNORA73 and snR30. Homology between these families has been established in the published literature [1,2]. Researchers at the human snoRNA database (snoRNABase) have established that the following human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: SNORA73 and snR30 [3].'),
('CL00079', 'snR68', NULL, 'snR68 clan', 'Gardner PP', 'The snR68 clan contains the RNA families snR68 and snoR27. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoR27 and SnR68 [1].'),
('CL00080', 'snoR53', NULL, 'snoR53 clan', 'Gardner PP', 'The snoR53 clan contains the RNA families snoR53 and snoR53Y. Researchers at the plant snoRNA database have established that the following arabadopsis, human and S. cerevisiae snoRNAs target modifications on orthologous positions of the ribosome: snoR53Y and SnR53 [1].'),
('CL00081', 'snoU13', NULL, 'snoU13 clan', 'Gardner PP', 'The snoU13 clan contains the RNA families snoZ279_R105_R108 and snoU13. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [1,2].'),
('CL00082', 'snoU85', NULL, 'snoU85 clan', 'Gardner PP', 'The snoU85 clan contains the RNA families snoU89 and snoU85. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00083', 'mir-6', NULL, 'mir-6 clan', 'Gardner PP', 'The mir-6 clan contains the RNA families mir-6 and mir-11. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [4]. This clan had a previous ID of mir-2 however this family was removed from the clan and the ID updated.'),
('CL00084', 'mir-3', NULL, 'mir-3 clan', 'Gardner PP', 'The mir-3 clan contains the RNA families mir-3 and mir-318. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH and PRC [1,2]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [3].'),
('CL00085', 'mir-15', NULL, 'mir-15 clan', 'Gardner PP', 'The mir-15 clan contains the RNA families mir-16 and mir-15. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [4].'),
('CL00086', 'mir-28', NULL, 'mir-28 clan', 'Gardner PP', 'The mir-28 clan contains the RNA families mir-28 and mir-708. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [4].'),
('CL00087', 'mir-34', NULL, 'mir-34 clan', 'Gardner PP', 'The mir-34 clan contains the RNA families mir-34 and mir-449. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [1,2]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [3].'),
('CL00088', 'mir-36', NULL, 'mir-36 clan', 'Gardner PP', 'The mir-36 clan contains the RNA families mir-35, mir-36 and mir-42. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [4].'),
('CL00089', 'mir-50', NULL, 'mir-50 clan', 'Gardner PP', 'The mir-50 clan contains the RNA families mir-190 and mir-50. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH and PRC [1,2]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [3].'),
('CL00090', 'mir-73', NULL, 'mir-73 clan', 'Gardner PP', 'The mir-73 clan contains the RNA families mir-74 and mir-73. The following bioinformatic methods were used to confirm the relationship between these families: PRC [1]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [2].'),
('CL00091', 'mir-81', NULL, 'mir-81 clan', 'Gardner PP', 'The mir-81 clan contains the RNA families bantam and mir-81. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [4].'),
('CL00092', 'mir-137', NULL, 'mir-137 clan', 'Gardner PP', 'The mir-137 clan contains the RNA families mir-137 and mir-234. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [4].'),
('CL00093', 'mir-182', NULL, 'mir-182 clan', 'Gardner PP', 'The mir-182 clan contains the RNA families mir-183, mir-182, mir-263 and mir-228. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [4].'),
('CL00094', 'mir-216', NULL, 'mir-216 clan', 'Gardner PP', 'The mir-216 clan contains the RNA families mir-216 and mir-283. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [1,2]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [3].'),
('CL00095', 'mir-279', NULL, 'mir-279 clan', 'Gardner PP', 'The mir-279 clan contains the RNA families mir-279 and mir-996. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [4].'),
('CL00096', 'mir-290', NULL, 'mir-290 clan', 'Gardner PP', 'The mir-290 clan contains the RNA families mir-515, mir-290, mir-302 and miR-430. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [1,2]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [3].'),
('CL00097', 'mir-BART', NULL, 'mir-BART clan', 'Gardner PP', 'The mir-BART clan contains the RNA families mir-BART1 and mir-BART3. The following bioinformatic methods were used to confirm the relationship between these families: PRC [1]. The hexamer seeds of the mature miRNA sequences from these families were manually checked for similarity using sequences obtained from miRBase [2].'),
('CL00098', 'MIR169_2', 'MIR169;', 'MIR169 clan', 'Gardner PP', 'The MIR169 clan contains the RNA families MIR169_2 and MIR169_5. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00099', 'MIR171', NULL, 'MIR171 clan', 'Gardner PP', 'The MIR171 clan contains the RNA families MIR171_1 and MIR171_2. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3].'),
('CL00100', 'U3', NULL, 'U3 clan', 'Gardner PP', 'The U3 clan contains the U3 RNA families specific to metazoa, fungi, plants and basal eukaryotes. The homology and divisions between these families has been established in the published literature [1].'),
('CL00101', 'Cobalamin', NULL, 'Cobalamin clan', 'Daub J', 'The Cobalamin clan contains the RNA families for Cobalamin riboswitches and the AdoCbl riboswitch from Enterobacteria. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH [1]. The relationship and distinction between these RNA families has been noted in the published literature [2,3].'),
('CL00102', 'group-II-D1D4', NULL, 'Group II catalytic intron D1-D4 clan', 'Gardner PP', 'The group II catalytic intron D1-D4 clan contains the diverse RNA families from the 5` region of group II catalytic intron. Homology between these families has been established in the published literature [1-6].'),
('CL00103', 'SNORD11', NULL, 'SNORD11 clan', 'Gardner PP', 'The SNORD11 clan contains the RNA families SNORD11 and SNORD11B. The following bioinformatic methods were used to confirm the relationship between these families: HMMER3 and PRC [1,2].'),
('CL00104', 'Hepadnavirus_epsilon', NULL, 'Hepadnavirus_epsilon clan', 'Brown C', 'The Hepadnavirus_epsilon clan contains the RNA families HBV_epsilon and AHBV_epsilon. Homology between these families has been established in the published literature [1-4].'),
('CL00105', 'RyeA-RyeB', 'RF00111;', 'RyeA/RyeB clan', 'Burge SW', 'The RyeA/RyeB clan contains the RNA families RyeA and RyeB. These two families overlap as part of a bidirectional sRNA locus in E. coli'),
('CL00106', 'Csr_Rsm_clan', NULL, 'Csr-Rsm protein binding clan', 'Gardner PP', 'The Csr-Rsm clan contains the sRNA families CsrC, RsmB, RsmX, RsmY, RsmZ and TwoAYGGAY. These families are bound by multiple regulatory proteins called CsrA or RsmA. These bind to hairpins containing the sequence motif \'GGA\'. Homology between these families has been established in the published literature [1-9].'),
('CL00108', 'suhB', NULL, 'suhB sRNA clan', 'Eberhardt R', 'This clan includes the suhB and ar15 sRNAs. The homology between these families is restricted to the second hairpin and Rho-independent terminator of suhB [1].'),
('CL00110', 'mir-19', NULL, 'mir-19 clan', 'Burge SW', 'The mir-19 clan contains the RNA families mir-19 and mir-363. The following bioinformatic methods were used to confirm the relationship between these families: CMSEARCH, HMMER3 and PRC [1,2,3]. '),
('CL00111', 'SSU', NULL, 'SSU clan', 'Burge SW', 'This clan contains the SSU families.'),
('CL00112', 'LSU', NULL, 'LSU clan', 'Burge SW', 'This clan contains the LSU families.'),
('CL00113', '5S_rRNA', NULL, '5S rRNA clan', 'Nawrocki EP', 'Includes 5S rRNA family with permuted secondary structure.'),
('CL00114', 'LhrC', NULL, 'LhrC clan', 'Argasinska J', 'The LhrC clan contains the RNA families LhrC, Rli22 and Rli33, which are structurally and functionally related. They contain UCCC motifs important for post-transcriptional repression.[1]'),
('CL00115', 'DUF805', NULL, 'DUF805 clan', 'Weinberg Z', 'The DUF805 clan contains the RNA families DUF805 and DUF805b.'),
('CL00116', 'Coronavirus-5UTR', 'Coronavirus;', 'Coronavirus 5\' UTR', 'Petrov AI', 'The Coronavirus clan groups structured RNA elements found in Coronavirus genomes.'),
('CL00117', 'Coronavirus-3UTR', 'Coronavirus;', 'Coronavirus 3\' UTR', 'Petrov AI', 'The Coronavirus clan groups structured RNA elements found in Coronavirus genomes.'),
('CL00118', 'L31_rleader', NULL, 'Ribosomal Protein L31 Leader clan', 'Weinberg Z; 0000-0002-6681-3624', 'The L31 r-leaders are five motifs that likely bind the L31 r-protein, they are used as an autoregulatory mechanism to control the concentration of the ribosomal protein L31 [1]'),
('CL00119', 'S4_rleader', NULL, 'Ribosomal Protein S4 Leader clan', 'Weinberg Z; 0000-0002-6681-3624', 'The S4 rleaders are motifs that likely bind the S4 r-protein, they have potential to imitate the rRNAs binding site (GUAA sequence). rleaders has been propose participate in an autoregulatory mechanism to control the concentration of the ribosomal proteins [1]'),
('CL00120', 'twister_ribozyme', NULL, 'twister ribozyme', 'Weinberg Z; 0000-0002-6681-3624', 'Twister ribozyme (Type-P1) RF03160 and variants (Type-P3 and Type-P5) RF03154 and RF02684 have been reported in eukaryotic and bacterial genomes. They are comparable in size and structural complexity to riboswitches and their genetic contexts often bear marked similarities to those of hammerhead ribozymes [1]'),
('CL00121', 'S15_rleader', NULL, 'Ribosomal Protein S15 Leader clan', 'Weinberg Z; 0000-0002-6681-3624', 'The Ribosomal Protein S15 Leader clan includes RF00114, RF03137, RF03143 and RF0 families, they are RNA motifs used as an autoregulatory mechanism to control the concentration of the ribosomal protein S15 [1,2]. S15 rleaders have been reported in bacteria and in archaea [2]'),
('CL00122', 'NiCo_riboswitch', NULL, 'NiCo_riboswitches', 'Weinberg Z; 0000-0002-6681-3624', 'Nico riboswitches are selective metalloregulatory riboswitches [1], they bind four divalent metal ions, and use to be upstream of genes encoding transporters of Co2+, Zn2+ and Cd2+ [2]'),
('CL00123', 'Purine_riboswitch', NULL, 'Purine riboswitches', 'Weinberg Z; 0000-0002-6681-3624', 'Riboswitches are metabolite binding domains within certain messenger RNAs that serve as precision sensors for their corresponding targets. Allosteric rearrangement of mRNA structure is mediated by ligand binding, and this results in modulation of gene expression. A class of riboswitches that selectively recognises guanine has been reporter first in [1]. It has also been shown that Purine riboswitch variants of this family are specific for adenine [2], hypoxanthine [3] and 2\'-deoxyguanosine [6].'),
('CL00124', 'TD', NULL, 'TD (Treponema denticola) motifs', 'Weinberg Z; 0000-0002-6681-3624', 'Clan TD include two Treponema dentricola motifs, TD-1 and TD-2, in some cases TD-1 motif overlap reverse complements of TD-2 motif. TD-1 is usually in 5\' regulatory region and it has been proposed as a cis-regulatory element while TD-2 is not in 5\' regulatory regions and has been propose as a non-coding RNA '),
('CL00125', 'Glycine_riboswitch', NULL, 'Glycine riboswitch clan', 'Weinberg Z; 0000-0002-6681-3624', 'Glycine riboswitches clan include the glycine riboswitch first reported in [1] and the glycine-GGAnGA variant which is particular in genes that encode either saccharopine dehydrogenase or amino acid transporters and other genes with not predicted function [2].'),
('CL00126', 'c-di-GMP_I_riboswitch', NULL, 'c-di-GMP_I riboswitch clan', 'Weinberg Z; 0000-0002-6681-3624', 'Riboswitches are metabolite binding domains within certain messenger RNAs that serve as precision sensors for their corresponding targets. Allosteric rearrangement of mRNA structure is mediated by ligand binding, and this results in modulation of gene expression. A class of riboswitches that selectively recognises cyclic dinucleotides c-di-GMP and c-di-AMP was report first in [2]. c-di-GMP_I riboswitch clan included the first reported c-di_GMP_I riboswitch and the variants c-di-GMP-I-UAU and c-di-GMP-I-GGC reported in [1].'),
('CL00127', 'c-di-GMP_II_riboswitch', NULL, 'c-di-GMP_II riboswitch clan', 'Weinberg Z; 0000-0002-6681-3624', 'Clan c-di-GMP_II riboswitch included the first reported c-di_GMP_II riboswitch [2] and two variants c-di-GMP-II-GAG and c-di-GMP-II-GCG reported in [1]'),
('CL00128', 'C4-antisense-RNA', NULL, 'C4-antisense-RNA clan', 'Weinberg Z; 0000-0002-6681-3624', 'The c4 antisense RNA clan include c4 antisense RNA RF01695, variant c4-2 antisense RNA RF02944 and variant c4-a1b1 antisense RNA RF03163. c4 antisense RNAs are used by phagues to inhibit the synthesis of antirepressor genes. It is proposed that c4 antisense RNAs are responsible for heteroimmunity in phagues.'),
('CL00129', 'Flavivirus-5UTR', 'SLA_Flavivirus;', 'Flavivirus 5\' UTR clan', 'Lamkiewicz K; 0000-0002-6375-6441', 'The Flavivirus 5\'-SLA clan contains the RNA families: RF02340 SLA_Flavivirus and RF03546 Flavivirus-5UTR'),
('CL00131', 'Flavi_ISFV_repeat_Ra_Rb', NULL, 'Insect-Specific Flavivirus 3\' UTR repeats Ra and Rb elements', 'Lamkiewicz K; 0000-0002-6375-6441', '3\' UTRs in insect specific flavivirus families present three to four repeats of conserved element Ra and Rb [1]. Can happend that only Ra or Rb element is present.'),
('CL00132', 'Archaea_CD-box_snoRNA', NULL, 'Archaeal small nucleolar RNA CD-box', 'Ontiveros-Palacios N; 0000-0001-8457-4455', 'This Clan include Rfam:RF00065 Small nucleolar RNA snoR9 reported in a computational screen in AT-rich hyperthermophiles [3], Rfam:RF00095 smaller snoRNA R19 a member of Pyrococcus C/D box snoRNA family [1,2], Rfam:RF01131 snoRNA SR47 and Rfam:RF01309 snoRNA SR60 [1].'),
('CL00133', 'mir-31', NULL, 'mir-31 microRNA precursor family', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'The mir-31 microRNA clan contains the RNA families RF00661_mir-31 and RF04091_mir-72'),
('CL00134', 'mir-31', NULL, 'mir-31 microRNA precursor family', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'The mir-31 microRNA clan contains the RNA families RF00661_mir-31 and RF04091_mir-72'),
('CL00135', 'mir-252', NULL, 'mir-252 microRNA precursor family', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'The mir-252 microRNA clan contains the RNA families RF00837_mir-251 and RF00838_mir-252'),
('CL00136', 'mir-465', NULL, 'mir-465 microRNA precursor family', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'The mir-252 microRNA clan contains the RNA families RF03464_mir-465 and'),
('CL00137', 'mir-92', NULL, 'mir-92 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'mir-92 clan contains the families mir-92 and mir-310'),
('CL00138', 'mir-8', NULL, 'mir-8 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'mir-8 clan contains the families mir-8 and mir-236'),
('CL00139', 'MIR2863', NULL, 'MIR2863 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'MIR2863 clan contains the families MIR2863 and MIR5070'),
('CL00140', 'mir-506', 'mir-125;', 'mir-506 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'mir-506 clan contains the families mir-506 and mir-511'),
('CL00141', 'mir-H7', 'MIR167; mir-743', 'mir-H microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'mir-H7 clan contains the families mir-H7 and mir-H20'),
('CL00142', 'mir-2733', 'mir-64;', 'mir-2733 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'mir-2733 clan contains the families mir-2733 and mir-2733'),
('CL00143', 'MIR827', NULL, 'MIR827 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'MIR827 clan contains the families MIR827 and MIR827_5'),
('CL00144', 'MIR4387', NULL, 'MIR4387 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'MIR4387 clan contains the families MIR4387 and MIR7486'),
('CL00145', 'MIR1520', NULL, 'MIR1520 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'MIR1520 clan contains the families MIR1520 and MIR4372'),
('CL00146', 'mir-8791', NULL, 'mir-8791 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'mir-8791 clan contains the families mir-8791 and mir-8799'),
('CL00147', 'mir-574', 'mir-509;', 'mir-574 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'mir-574 clan contains the families mir-574 and mir-9195'),
('CL00148', 'let-7', NULL, 'let-7 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'let-7 clan contains the families let-7 and mir-3596'),
('CL00149', 'mir-154', NULL, 'mir-154 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'mir-154 clan contains the families mir-154, mir-1197, mir-368, mir-329, mir-379, mir-485, mir-889, mir-3578'),
('CL00150', 'MIR162', NULL, 'MIR167 microRNA precursor clan', 'Griffiths-Jones SR; 0000-0001-6043-807X', 'MIR162 clan contains the families MIR162_1 and MIR162_2');

--
-- Indexes for dumped tables
--

--
-- Indexes for table `clan`
--
ALTER TABLE `clan`
  ADD PRIMARY KEY (`clan_acc`);
COMMIT;

/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
